homeassistant/components/mikrotik/hub.py

```diff
@@ -96,6 +97,7 @@ def __init__(self, hass, config_entry, api):
         self.devices = {}
         self.available = True
         self.support_wireless = bool(self.command(MIKROTIK_SERVICES[IS_WIRELESS]))
+        self.support_capsman = bool(self.command(MIKROTIK_SERVICES[IS_CAPSMAN]))
```

Comment:
We want to avoid side effects in init method. We should move this outside after instantiating the class or into a dedicated setup method.

Reply:
moved it to only method using the variable